packages/services/f-http/src/index.js

```diff
@@ -5,8 +5,8 @@ import httpVerbs from './httpVerbs';
 /**
  *  @class httpModule
  *  @type {Object}
- *  @property {function} createClient Create a new HTTP client - accepts options.
- *  @property {mockFactory} mockFactory Exposes functions to setup mocking of the underlying HTTP connection
+ *  @property {object} CreateClient Create a new HTTP client - accepts options.
```

Comment:
Can we define what options are valid?

Reply:
These are documented in the readme; thats a pattern we use for all of the services